ElectronClient/bridge.js

```diff
@@ -161,6 +161,10 @@ class Bridge {
 		return require('electron').screen;
 	}
 
+	// Detect system theme
+	systemTheme() {
+		return nativeTheme.shouldUseDarkColors; // return true for system dark theme
+	}
```

Comment:
We have a nicely named function `shouldUseDarkColors` and you create a wrapper and rename it to `systemTheme`, which is meaningless. Is it a boolean, is it a string? What are the possible values? Please just keep the value `shouldUseDarkColors`. And please remove your comments as they provide no info.

Reply:
okay I'll change wrapper name to `shouldUseDarkColors`